docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -0,0 +1,150 @@
+
+### Step 1: Clone the Snapshotter Deploy Repository
+
+Clone the repository against the respective branch (main by default). Open the terminal and run the below command to clone the deploy repo in a directory named `powerloom-deploy`.
+
+```bash
+git clone https://github.com/PowerLoom/deploy.git --single-branch powerloom_deploy --branch devnet && cd powerloom_deploy
+```
+
+### Step 2: Fork the Snapshotter Computes and Snapshotter Config Repositories
+
+Our system leverages the Git submodule architecture to seamlessly manage these components. To dive deeper into how these elements integrate and function within our larger system, check out our [architecture documentation](https://docs.powerloom.io/docs/build-with-powerloom/snapshotter-node/architecture).
+To optimize your development process, we recommend forking the templates for:
+
+- Snapshotter Configs: https://github.com/PowerLoom/snapshotter-computes
+- Snapshotter Computes: https://github.com/PowerLoom/snapshotter-configs 
+
+Once you have forked the above repositories, next step is to clone them on your local system as well. 
+
+```bash
+git clone https://github.com/<your_github_username>/snapshotter-computes
+```
+```bash
+git clone https://github.com/<your_github_username>/snapshotter-configs
+```
+
+### Step 3: Configure the environment variables
+
+1. In the deploy repo's directory, create a new file named `.env`.
+
+2. Copy the contents of `env.example` to `.env`.
+
+3. Fill in the following required variables in the `.env` file:
+
+   - `SOURCE_RPC_URL`: The URL for Source RPC (Local node/Infura/Alchemy) service.
+
+   - `SIGNER_ACCOUNT_ADDRESS`: The address of the signer account. This is your burner wallet address added on the Devnet dashboard. **Using a burner account is highly recommended**. 
+
+   - `SIGNER_ACCOUNT_PRIVATE_KEY`: The private key corresponding to the burner wallet address.
+   - `SNAPSHOT_CONFIG_REPO`: https://github.com/your-username/snapshotter-configs [Change the username to your GitHub profile to point to your forked snapshotter-configs repository.]
+   - `SNAPSHOT_CONFIG_REPO_BRANCH`: Change to your own branch or use the `devnet` branch if you are just getting started.
+   - `SNAPSHOTTER_COMPUTE_REPO`:  https://github.com/your-username/snapshotter-computes [Change the username to your GitHub profile to point to your forked snapshotter-computes repository.]
+   - `SNAPSHOTTER_COMPUTE_REPO_BRANCH`: Change to your own branch or use the `devnet` branch if you are just getting started.
+   - `PROST_RPC_URL`: `https://rpc-prost1d.powerloom.io` The URL for the Powerloom Protocol Chain RPC service.
+   - `PROTOCOL_STATE_CONTRACT`: The contract address for the protocol state.
+
+:::note
+We offer two types of Protocol State Contracts, each tailored to different Epoch Sizes, to cater to the unique complexities of your application:
```

Comment:
A lot of lines seem very GPT like tbh, better to keep it to the point. For example -
```
Currently, there are two variations of Protocol State Contracts available. One where [Epoch]<link to epoch explanation from docs> size is set to 1 and other where epoch size is set to 10 blocks.  This should allow developers to build and experiment with a variety of use cases. If you have any custom needs, reach out to the team <link>
```
Just a quick draft but you should get the idea.